vision/google/cloud/vision/face.py

```diff
@@ -91,11 +91,13 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.face.Emotions`
         :returns: Populated instance of `Emotions`.
         """
-        joy_likelihood = getattr(Likelihood, response['joyLikelihood'])
-        sorrow_likelihood = getattr(Likelihood, response['sorrowLikelihood'])
+        joy_likelihood = getattr(Likelihood, response.get('joyLikelihood'))
```

Comment:
1.  You definitely don't want this. Your `KeyError` will just become an `AttributeError` if `None` comes from `get()`
1.  Your fake enum in `Likelihood` is pretty useless here. If

    ```
    response['joyLikelihood'] == 'LIKELY'
    ```

    this code sets

    ```
    joy_likelihood = Likelihood.LIKELY
    ```

    but 

    ```
    Likelihood.LIKELY == 'LIKELY'
    ```

    so you should've just used the original